### ISSUE TITLE
All non-US national anthems fail to play

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#5]
```
In the interactive presentation, on the <audio>/<video> slide, selecting any 
national anthem other than "United States" produces no sound.
```

Original issue reported on code.google.com by `pkasting@chromium.org` on 22 Jun 2010 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
Issue 30 has been merged into this issue.
```

Original comment by `paulir...@google.com` on 24 Jun 2010 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
This was probably caused because of the missing http-content headers on the app 
engine side.
Although it has been fixed in the code I'll leave this open until we test it 
works live.
```

Original comment by `erne...@google.com` on 5 Aug 2010 at 10:21
- Changed state: **Accepted**
- Added labels: **Slides**

**Comment by GoogleCodeExporter** [comment]
Original comment by `v...@google.com` on 13 Aug 2010 at 11:49
- Added labels: **Milestone-3**

**Comment by GoogleCodeExporter** [comment]
```
Issue 35 has been merged into this issue.
```

Original comment by `paulir...@google.com` on 17 Aug 2010 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
Issue 43 has been merged into this issue.
```

Original comment by `paulir...@google.com` on 17 Aug 2010 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
original mp3 file was removed from the repository at some point.
trying to find the cause of the removal before restoring it.
```

Original comment by `erne...@google.com` on 30 Aug 2010 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
fixed with check-ins 624423c361, 9b49d99dcc, 1c1824ecde, 4b590b2aa9, bc5931157f
```

Original comment by `erne...@google.com` on 1 Sep 2010 at 1:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 210 has been merged into this issue.
```

Original comment by `ericbide...@html5rocks.com` on 25 Sep 2010 at 5:56
